slackbot/bot.py

```diff
@@ -7,22 +7,27 @@
 import time
 from glob import glob
 from six.moves import _thread
-from slackbot import settings
 from slackbot.manager import PluginsManager
+import slackbot.globals as globs
 from slackbot.slackclient import SlackClient
+from slackbot.utils import till_end, till_white
 from slackbot.dispatcher import MessageDispatcher
 
 logger = logging.getLogger(__name__)
 
 
 class Bot(object):
-    def __init__(self):
+    def __init__(self, settings, root):
```

Comment:
by passing settings, you can have bot.py in the brain instead of in each bot folder.
Root is the folder the python script is launched from, used for updating commands.